### PR TITLE
[FIX] l10n_latam_check: check number required if payment method is "New Third Party Checks"

### PR DESCRIPTION
**Pull request by pablohmontenegro**
Task Adhoc: 33137
Task Latam: 1073

Description of the issue/feature this PR addresses:
Check number is not required for payments when payment method "New Third Party Checks" but is needed to be required.

Current behavior before PR:
Check number is not required for payments when payment method "New Third Party Checks".

Desired behavior after PR is merged:
Check number is required for payments when payment method "New Third Party Checks".

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by vbe-odoo** [comment]
Hello @pablohmontenegro 
Thanks for the fix and for reporting the issue.

I've done the FUN testing and it LGTM.
@jco-odoo this is ready for technical review.

Thanks!
cc' @ren-odoo @fvz-odoo

File: addons/l10n_latam_check/views/account_payment_view.xml [hunk 1]
```diff
@@ -65,7 +65,7 @@
                     string='Check Number'
                     attrs="{
                         'invisible': [('payment_method_code', '!=', 'new_third_party_checks'), '|', ('payment_method_code', '!=', 'check_printing'), ('l10n_latam_manual_checks', '=', False)],
-                        'required': ['|', ('payment_method_code', '=', 'new_third_party_checks'), ('payment_method_code', '=', 'check_printing'), ('l10n_latam_manual_checks', '=', True)],
+                        'required': ['|', ('payment_method_code', '=', 'new_third_party_checks'), '&amp;',('payment_method_code', '=', 'check_printing'), ('l10n_latam_manual_checks', '=', True)],
```
Comment (jco-odoo) [inline addons/l10n_latam_check/views/account_payment_view.xml:68]:
```suggestion
                        'required': ['|', ('payment_method_code', '=', 'new_third_party_checks'), '&amp;', ('payment_method_code', '=', 'check_printing'), ('l10n_latam_manual_checks', '=', True)],
```

Comment (vbe-odoo) [reply addons/l10n_latam_check/views/account_payment_view.xml:68]: @pablohmontenegro can you help me with this suggested change?
Thanks!

Comment (pablohmontenegro) [reply addons/l10n_latam_check/views/account_payment_view.xml]: @vbe-odoo yes! ready! Thanks!

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**